tsconfig.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
this does not follow our usual tsconfigs, please take the one from the framework as baseline.